core/new-gui/src/app/workspace/component/product-tour/product-tour.component.ts

```diff
@@ -1,38 +1,162 @@
+    <br><br>
+    `,
+    placement: 'right',
+    title: 'Operator Panel',
+    preventScrolling: true
+  },
+  {
+    anchorId: 'texera-operator-panel',
+    content: `
+    <p>Drag <b>Source: Scan</b> operator and drop it to the workflow panel. </p>
+    <p><b>Source: Scan</b> is a operator that read records from a table one by one.</p>
+    <center><img src="../../../assets/Tutor_Intro_Drag_Srouce.gif" height="450" width="600"></center>
+    <br><br>
+    `,
+    title: 'Select Operator',
+    placement: 'right',
+    preventScrolling: true
+  },
+  {
+    anchorId: 'texera-property-editor-grid-container',
+    content: `
+    <p>This is the operator editor area where users can set the properties of the operator. </p>
+    <p>Now we want to edit the property of <b>Source: Scan</b> operator by typing <b>twitter_sample</b>
+    , which specifies the data we want to use.</p>
+    <center><img src="../../../assets/Tutor_Property_Sample.gif" height="260" width="500"></center>
+    <br><br>
+    `,
+    placement: 'left',
+    title: 'Property Editor',
+    preventScrolling: true
+  },
+  {
+    anchorId: 'texera-operator-panel',
+    content: `<p>Now we want to view the results of selected data. Open <b>View Results</b> section.</p>
+    <center><img src="../../../assets/Tutor_OpenResult_Sample.gif" height="270" width="500"></center>
+    <br><br>
+    `,
+    placement: 'right',
+    title: 'Operator Panel',
+    preventScrolling: true
+  },
+  {
+    anchorId: 'texera-operator-panel',
+    content: `
+    <p>Drag <b>View Results</b> operator and drop it to the workflow panel.</p>
+    <center><img src="../../../assets/Tutor_Intro_Drag_Result.gif" height="400" width="590"></center>
+    <br><br>
+    `,
+    placement: 'right',
+    title: 'Select Operator',
+    preventScrolling: true
+  },
+  {
+    anchorId: 'texera-property-editor-grid-container',
+    content: `
+    <p>Connect those two operators.</p><center>
+    <img src="../../../assets/Tutor_JointJS_Sample.gif" height="150" width="500"></center>
+    <br><br>
+    `,
+    placement: 'left',
+    title: 'Connecting operators',
+    preventScrolling: true
+  },
+  {
+    anchorId: 'texera-workspace-navigation-run',
+    content: `
+    <p>Click the run button to execute the workflow.</p>
+    `,
+    title: 'Running the workflow',
+    placement: 'bottom',
+    preventScrolling: true
+  },
+  {
+    anchorId: 'texera-result-view-grid-container',
+    content: `
+    <p>You can view the results here.</p>
+    `,
+    placement: 'top',
+    title: 'Viewing the results',
+    preventScrolling: true
+  },
+  {
+    anchorId: 'texera-navigation-grid-container',
+    content: `
+    <center><h3>Congratulations!</h3></center>
+    <p>You have finished the basic tutorial. </p>
+    <p>There are many other operators you can use to form a workflow.</p>
+    <center><img src="../../../assets/Tutor_End_Sample.gif" height="335" width="640"></center>
```

Comment:
<p>You have finished the basic tutorial. </p>
    <p>There are many other operators that you can use to form a workflow.</p>